src/api/record/record.ts

```diff
@@ -28,9 +27,8 @@ export class RecordHandler extends RupertPlugin {
     private database: RecordDatabase = {}
   ) {
     super();
-    this.cancelWrite = setInterval(() => this.write(), 1 * 1000);
     readFile(this.dbPath, 'utf-8', (err: any, persisted: string) => {
-      if ( err !== null ) { return; }
+      if ( err ) { return; }
```

Comment:
We should probably log at `info` that we didn't find a database, and are starting a new one.